### ISSUE TITLE
slices

### ISSUE THREAD

**Issue by nrpr93** [despoisj/DeepAudioClassification#3]
When I use this command "python main.py slice", the code doesnt generate the slices, and for that reason I can train the net.

nrpr93@nrpr93-VirtualBox:~/Desktop/DeepAudioClassification-master$ python main.py slice
--------------------------
| ** Config ** 
| Validation ratio: 0.3
| Test ratio: 0.1
| Slices per genre: 1000
| Slice size: 128
--------------------------
Creating spectrograms...
Creating spectrogram for file 1/8...
Creating spectrogram for file 2/8...
Creating spectrogram for file 3/8...
Creating spectrogram for file 4/8...
Creating spectrogram for file 5/8...
Creating spectrogram for file 6/8...
Creating spectrogram for file 7/8...
Creating spectrogram for file 8/8...
Spectrograms created!
Creating slices...
Slices created!
nrpr93@nrpr93-VirtualBox:~/Desktop/DeepAudioClassification-master$ python main.py train
--------------------------
| ** Config ** 
| Validation ratio: 0.3
| Test ratio: 0.1
| Slices per genre: 1000
| Slice size: 128
--------------------------
Traceback (most recent call last):
  File "main.py", line 37, in <module>
    genres = os.listdir(slicesPath)
OSError: [Errno 2] No such file or directory: 'Data/Slices/'

**Comment by despoisj** [comment]
It looks like you don't have created the Slices folder

2017-05-30 12:59 GMT+02:00 Nelson Ricardo Pereira Rodrigues <
notifications@github.com>:

**Comment by nrpr93** [comment]
I already create folder but still doesn´t work

**Comment by despoisj** [comment]
Then do you still have the error ? Did it change to another error ?

2017-05-30 18:59 GMT+02:00 Nelson Ricardo Pereira Rodrigues <
notifications@github.com>:

**Comment by nrpr93** [comment]
no, the main problem, is this: when I slice the musics in the data/raw folders, your implementation should create de slices inside the SLices folder, the problem is that not happen, but when I use the slice command don´t return error, it says the slices are created, but they aren´t. And for this reason I can´t train.

**Comment by papshpapsh** [comment]
I also experienced this problem today, and the reason was in sox, it didn't support mp3. After installing libid3tag all started to work. Thanks!
